plugins/hls-stan-plugin/src/Ide/Plugin/Stan.hs

```diff
@@ -162,24 +139,21 @@ rules recorder plId = do
 
               -- If envVar is set to 'False', stan will ignore all local and global .stan.toml files
               logWith recorder Debug (LogDebugStanEnvVars env)
-              seTomlFiles <- liftIO $ usedTomlFiles useDefConfig (stanArgsConfigFile stanArgs)
 
               (cabalExtensionsMap, checksMap, confIgnored) <- case configTrial of
                   FiascoL es -> do
                       logWith recorder Development.IDE.Warning (LogWarnConf es)
                       pure (Map.empty,
                             HM.fromList [(LSP.fromNormalizedFilePath file, inspectionsIds)],
                             [])
-                  ResultL warnings stanConfig -> do
+                  ResultL _warnings stanConfig -> do
                       let currentHSAbs = fromNormalizedFilePath file -- hie_hs_file hie
                       currentHSRel <- liftIO $ makeRelativeToCurrentDirectory currentHSAbs
                       cabalExtensionsMap <- liftIO $ createCabalExtensionsMap isLoud (stanArgsCabalFilePath stanArgs) [hie]
 
                       -- Files (keys) in checksMap need to have an absolute path for the analysis, but applyConfig needs to receive relative
                       -- filepaths to apply the config, because the toml config has relative paths. Stan itself seems to work only in terms of relative paths.
                       let checksMap = HM.mapKeys (const currentHSAbs) $ applyConfig [currentHSRel] stanConfig
-
-                      let analysis = runAnalysis cabalExtensionsMap checksMap (configIgnored stanConfig) [hie]
```

Comment:
Not sure if this should be removed, but `analysis` - result of this pure computation is unused below, so seems pointless to compute.

Reply:
Yes, it seems I forgot to delete that line when iterating with this function.